diesel_cli/src/print_schema.rs

```diff
@@ -144,6 +144,8 @@ fn mysql_diesel_types() -> HashSet<&'static str> {
 
     types.insert("TinyInt");
     types.insert("Tinyint");
+    types.insert("Datetime");
```

Comment:
Does it make sense to add `TIMESTAMP` along with it?

Reply:
It's already there: https://github.com/diesel-rs/diesel/blob/b51e6584ac4c6dc4495010e6e2f282c6de327e1f/diesel_cli/src/print_schema.rs#L79